src/Data/Hints.cs

```diff
@@ -59,6 +61,7 @@ public struct HeroGraveHint {
         public static Dictionary<string, HeroGraveHint> HeroGraveHints = new Dictionary<string, HeroGraveHint>();
 
         public static Dictionary<string, string> HintMessages = new Dictionary<string, string>();
+        public static Dictionary<string, int> StartInventoryItems = new Dictionary<string, int>();
```

Comment:
I don't think this needs to be a global variable does it?

Reply:
It's getting populated by the Archipelago stuff, so it should be a global